### ISSUE TITLE
Use the preferred file editor configured in git?

### ISSUE THREAD

**Issue by foucdeg** [williamdclt/git-pretty-pull-request#61]
Something like `EDITOR=$(git config --get core.editor)`

**Comment by foucdeg** [comment]
@williamdclt After reading more on this, it seems that the env variable you used, `EDITOR`, is a system-wide standard, which git actually uses if you don't specify `core.editor`.
So I guess I could `export EDITOR=whatever` too.

**Comment by williamdclt** [comment]
Yeah I have `export EDITOR=vim` in my .zshrc :) But using the git editor definitely makes sense, a PR would be welcome

**Comment by foucdeg** [comment]
which do you think should take precedence between `EDITOR` and `git config --get core.editor` ?

**Comment by williamdclt** [comment]
git config --get core.editor, like in git itself
